src/prop_types.js

```diff
@@ -150,3 +150,20 @@ const layoutDangerousStyles = everythingDangerousStyles;
 const layoutItemDangerousStyles = everythingDangerousStyles.concat([
   "position", "margin", "marginTop", "marginBottom", "marginLeft", "marginRight"
 ]);
+
+
+export const cleanProps = function (props) {
+  const proprietaryProps = Object.keys(Object.assign({},
+    HLayoutPropTypes,
+    HLayoutDefaultPropTypes,
+    HLayoutItemPropTypes,
+    VLayoutPropTypes,
+    VLayoutDefaultPropTypes,
+    VLayoutItemPropTypes
+  ));
+  let safeProps = Object.assign({}, props);
```

Comment:
Alright I will do so. I wasn't sure because I saw a ticket or something about removing lodash but I prefer to use it in fact